packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -104,4 +106,22 @@ router.get(
   })
 );
 
+// BELOW ARE THE ROUTES PERTAINING TO THE IHE-GATEWAY
+
+/**
+ * POST /internal/carequality/patient-discovery
+ *
+ * Receives a Patient Discovery request from the IHE Gateway
+ */
+router.post(
+  "/patient-discovery",
+  asyncHandler(async (req: Request, res: Response) => {
+    const patientDiscovery = patientDiscoveryResponseSchema.parse(req.body);
+
+    await handleDiscoverResponse(patientDiscovery);
```

Comment:
This endpoint is called by Mirth, right? Do we need to await here, or can we just trigger the Promise and return OK so Mirth doesn't have to wait for it?

Reply:
Yea thats a good point its triggered by Mirth and its not like it expects anything